src/GraphODataTemplateWriter/Extensions/OdcmModelExtensions.cs

```diff
@@ -42,7 +42,7 @@ private static OdcmNamespace GetOdcmNamespace(OdcmModel model)
         public static IEnumerable<OdcmClass> GetComplexTypes(this OdcmModel model)
         {
             var @namespace = GetOdcmNamespace(model);
-            return @namespace.Classes.Where(x => x is OdcmComplexClass);
+            return @namespace.Classes.Where(x => x is OdcmComplexClass && x.CanonicalName() != "microsoft.graph.Json");
```

Comment:
LOL. I agree and this comment was made. I actually probably should do a case insensitive check here.